content/en/tracing/llm_observability/sdk.md

```diff
@@ -596,11 +601,79 @@ def process_message():
     return
 {{< /code-block >}}
 
+
+### Distributed tracing
+
+The SDK supports tracing across distributed services or hosts.
+
+#### Automatic distributed tracing
```

Comment:
Suggestion for the way this section should be phrased/structured

***
The SDK supports tracing across distributed services or hosts. Distributed tracing works by propagating span information across web requests.

If your application makes requests using these supported web framework and http libraries, you can enable distributed tracing by running:
ddtrace.patch(<INTEGRATION_NAME>)

If your application does not use any of these supported libraries, you can enable distributed tracing by manually propagating span information from http headers.
.... 
***

Reply:
Mainly because this clarifies what is exactly being "manually" vs "automatically" done (injecting and extracting from request headers)